test/Atc.Rest.ApiGenerator.Tests/SyntaxGenerators/Api/EndpointControllersTest.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Atc.Rest.ApiGenerator.Helpers;
+using Atc.Rest.ApiGenerator.Models;
+using Atc.Rest.ApiGenerator.SyntaxGenerators;
+using Atc.Rest.ApiGenerator.SyntaxGenerators.Api;
+using Atc.Rest.ApiGenerator.Tests.Helpers.CodeGenerator;
+using VerifyXunit;
+using Xunit;
+
+namespace Atc.Rest.ApiGenerator.Tests.SyntaxGenerators
+{
+    [UsesVerify]
+    public class EndpointControllersTest : SyntaxGeneratorTestBase
```

Comment:
```suggestion
    public class SyntaxGeneratorEndpointControllersTest : SyntaxGeneratorTestBase
```
and rename filename to match class